library/axi_dmac/axi_dmac_constr.ttcl

```diff
@@ -172,11 +172,6 @@ set_false_path -quiet \
 	-to [get_pins -quiet -hier *reset_sync_in_reg*/PRE \
 		-filter {NAME =~ *i_reset_manager*}]
 
-set_property -dict { \
-    SHREG_EXTRACT NO \
-    ASYNC_REG TRUE \
-  } [get_cells -quiet -hier *reset_shift_reg*]
```

Comment:
Should be renamed to reset_async_reg

Reply:
I thought reset_shift_reg was renamed to reset_sync_reg, but obviously it was split into sync and async part.

Thanks.